cmd/imagec/docker.go

```diff
@@ -144,16 +145,17 @@ func FetchToken(url *url.URL) (*Token, error) {
 }
 
 // FetchImageBlob fetches the image blob
-func FetchImageBlob(options ImageCOptions, image *ImageWithMeta) error {
+func FetchImageBlob(options ImageCOptions, image *ImageWithMeta) (*string, error) {
```

Comment:
I just didn't like returning "" in the error cases. I'm happy to change it if it's not proper go convention.